src/surveys/components/TextResponse.tsx

```diff
@@ -0,0 +1,87 @@
+import { h } from 'preact';
+import { CTASettings, Question, TextSettings, Theme } from '../../types';
+import { useState } from 'preact/hooks';
+import { Button, Header } from '../../components';
+type Props = {
+  question: Question;
+  theme?: Theme;
+  onAnswered: (questionId: string | number, answer: string) => void;
+};
+
+const TextResponse = ({ theme, question, onAnswered }: Props) => {
+  const [answer, setAnswer] = useState('');
+
+  const onSubmitHandler = (
+    event: h.JSX.TargetedEvent<HTMLFormElement, Event>
+  ) => {
+    event.preventDefault();
+    if (onAnswered) {
+      onAnswered(question?.id, answer);
+    }
+  };
+
+  return (
+    <div className={'max-w-[440px]'}>
+      <Header
+        title={question.label}
+        description={question.description}
+        color={theme?.question}
+      />
+      <div>
+        <form onSubmit={onSubmitHandler}>
+          {(question.settings as TextSettings).singleLine === true ? (
+            <input
+              type="text"
+              className={
+                'w-full mt-3 border border-gray-300 rounded-xl p-3 bg-formily-grey focus:outline-none focus:ring-2 focus:border-transparent'
+              }
+              style={{
+                color: theme?.question,
+                backgroundColor: theme?.answer,
```

Comment:
@afolasope The `color` should be `theme?.answer` and the `backgroundColor` should also be `theme?.answer` but with 10% opacity.